rna-transcription.json

```diff
@@ -0,0 +1,105 @@
+{
+   "#": [
+     "Language implementations vary on the issue of missing input variables.",
+     "In this set of test cases, the input variable is left out of the definition,",
+     "and the generator would need to handle the language implementation."
```

Comment:
Right, the transcription of an empty string should be an empty string.

How about `null`?